### DISCUSSION TITLE
Search issues with exact match

### DISCUSSION THREAD

**Discussion by r7kamura** [community/community#17956] [Projects and Issues] [answered]
Currently we can't search issues by exact match. It would be nice to be able to use exact match with "double quotes", like Google search.

For example, suppose we have issues with the following titles:

- 2022-06-01
- 2022-06-02
- 2022-06-03
- 2022-06-04
- 2022-06-05
- 2022-06-06

In this case, when searching with the following query, all issues match because of the "06" part.

```
in:title "2022-06-06"
```

**Comment by marii83** [comment, marked as answer]
in:title "2022-06-06"

**Reply by r7kamura** [reply]
Thank you! Sure worked well.

For some reason, I had tried it with single quote like `gh issue list --search '2022-06-06' in:title" ...` and apparently that's why it wasn't working properly.

**Reply by php-coder** [reply]
Unfortunately, it doesn't work for me:
```console
$ gh issue list --repo php-coder/mystamps --search "'Add client-side validation' in:title"

Showing 2 of 2 issues in php-coder/mystamps that match your search

#158  Add client side validation                     area/validation, idea              about 7 years ago
#23   Add client-side validation for e-mail address  area/validation, kind/improvement  about 8 years ago
$ gh version
gh version 2.30.0 (2023-05-30)
https://github.com/cli/cli/releases/tag/v2.30.0
```
